conf/hadoop2/spark-core-template.xml

```diff
@@ -0,0 +1,62 @@
+<configuration>
```

Comment:
I see. However there are other spark specific properties we need to add for Spark in addition to the ones in bigtable's hbase-site.xml. What's the best way to put them into spark's core-site.xml?